VideoLocker/res/values/strings.xml

```diff
@@ -425,4 +425,10 @@
 
     <!--  Email subject line when user attempts to submit feedback about the app -->
     <string name="email_subject">Customer Feedback</string>
+
+    <!-- Message indicating videos deleted -->
```

Comment:
Please describe the argument (the ``%s``) in the comment so it is clear to the translator what will be substituted. For example, ``Argument is number of videos deleted``